armi/resources/mc2Nuclides.yaml

```diff
@@ -1,1724 +0,0 @@
-AC225:
```

Comment:
There is a stray reference to this file in the armi/reactor/blueprints/isotoopicOptions module.

![image](https://user-images.githubusercontent.com/21959600/205785441-18005af2-ac6d-4170-8b9a-0f116aabbc7d.png)

Reply:
Fixed!